README.md

```diff
@@ -15,6 +15,17 @@ Or directly from our seed node:
 
     cargo install radicle-cli --force --locked --git https://seed.alt-clients.radicle.xyz/radicle-cli.git
 
+### 🐳 From Dockerfile
+
+Alternatively you can build the Radicle CLI docker `rad` by running the
+following command from inside this repository:
+
+    docker build docker build . -t rad
```

Comment:
Oh, perhaps this should be `docker build . -t rad` instead?

Reply:
Thanks - Oops fixed :)